package.json

```diff
@@ -36,6 +36,10 @@
   "engines": {
     "node": ">=8"
   },
+  "funding": {
+    "type": "Tidelift",
+    "url": "https://tidelift.com/subscription/pkg/npm-standard"
```

Comment:
How did you come up with these values, please?

Reply:
I just clicked on the GitHub **Sponsor** button and followed the Tidelift url from there, feel free to ignore this PR and implement whatever makes sense for the project 😊 I just wanted to raise awareness about this new `package.json` property that is now supported by **npm**

Reply:
Your PR is appreciated, @ruyadorno. As we do in programming, I would like to make sure that the values are correct. And I could not find the definitions for these fields. The best is the [rfc](https://github.com/npm/rfcs/blob/latest/accepted/0017-add-funding-support.md#implementation), in which there is no enumeration of the `type` field.

Also, following the `url` doesn't seem to let my easily donate. So I'm not sure it's the perfect URL. @feross would know.

Reply:
right right 👍 def sounds like it's not the best url to use in this case